doc-examples/example-groovy/src/main/groovy/example/ProfileService.groovy

```diff
@@ -16,16 +18,18 @@ import java.nio.file.StandardCopyOption
 @Singleton
 class ProfileService {
 
-    @Inject
-    ObjectStorageOperations objectStorage
+    final ObjectStorageOperations<PutObjectRequest.Builder, PutObjectResponse> objectStorage
+    ProfileService(ObjectStorageOperations<PutObjectRequest.Builder, PutObjectResponse> objectStorage) {
```

Comment:
I have used constructor injection because we always recommend users to use constructor injection when possible.